spec/truffle/launcher_spec.rb

```diff
@@ -24,7 +24,7 @@
                 truffleruby: /truffleruby .* like ruby 2\.3\.5/ }
 
   launchers.each do |launcher, (test, skip_success)|
-    bin_dirs = [RbConfig::CONFIG['bindir'], *RbConfig::CONFIG['extra_bindirs']]
+    bin_dirs = [File.dirname(RbConfig.ruby), *RbConfig::CONFIG['extra_bindirs']]
```

Comment:
This should be a separate commit, since it's addressing a somewhat different problem.

Reply:
yeah I was in a hurry